Resources/public/js/views/services/ez-navigationhubviewservice.js

```diff
@@ -53,6 +54,84 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
             });
         },
 
+        /**
+         * Fetches data about user
+         *
+         * @method _fetchUserInfo
+         * @param event {Object} event facade
+         */
+        _fetchUserInfo: function (event) {
+            var user = this.get('app').get('user'),
+                userService = this.get('capi').getUserService(),
+                userInfo = {},
+                promise = new Y.Promise(function (resolve, reject) {
+                    userService.loadUser(user.get('id'), Y.bind(function (error, response) {
+                        try {
+                            userInfo.userData = response.document.User;
+                            resolve(user.get('id'));
+                        } catch (e) {
+                            reject({
+                                message: 'Cannot get user data',
+                                identifier:  'fetch-user-info-' + Date.now()
+                            }, e);
+                        }
+                    }, this));
+                });
+
+            promise.then(function(userId) {
+                return new Y.Promise(function (resolve, reject) {
+                    userService.getRoleAssignmentsForUser(userId, Y.bind(function (error, response) {
+                        try {
+                            if (response.document.RoleAssignmentList && response.document.RoleAssignmentList.RoleAssignment[0]) {
+                                resolve(response.document.RoleAssignmentList.RoleAssignment[0].Role._href);
```

Comment:
actually getting the right role is a lot more complicated than that and even not really possible, because:
1. it's common to have several roles, and it's impossible to guess which one we should choose to display its name so that makes sense
1. the role is usually assigned to a group not to a user directly so in addition, so the same logic should be applied to groups
1. a user can be in several groups...

so IMHO, to decrease the complexity and avoid bugs we can't fix, we should skip that part and not display the user role(s) for now. If we really want that feature and we want it reliable, then we'll have to make changes server side.

@rolandbenedetti @inakijv @jince any opinion on that ?